.travis.yml

```diff
@@ -42,3 +42,5 @@ branches:
   only:
     - master
     - /^greenkeeper/.*$/
+    - labs
+    - /^lab/.*$/
```

Comment:
Maybe we should make it consistent:

`labs/base` as the template
`labs/<an-experimental-feature>` as the integration branch for `<an-experimental-feature>`